.github/workflows/build.yml

```
@@ -55,18 +49,103 @@ jobs:
             protobuf-c-compiler
             tcl
             uuid-dev
-            zlib1g-dev &&
+            zlib1g-dev
+        '
+      - name: Build from source
+        run: '
           (
-            mkdir original_comdb2/build &&
-            cd original_comdb2/build &&
+            mkdir bloomberg-comdb2/build &&
+            cd bloomberg-comdb2/build &&
             cmake .. &&
-            make &&
-            sudo make install
+            make
           )
         '
+      - name: Archive bloomberg-comdb2 repo with build artifacts
+        run: 'tar czvf bloomberg-comdb2.tar.gz bloomberg-comdb2/'
+      - name: Upload bloomberg-comdb2 repo with build artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: bloomberg-comdb2
+          path: ./bloomberg-comdb2.tar.gz
+
+  build_sdist:
+    name: Build python-comdb2 source distribution
+    runs-on: ubuntu-latest
+    needs: [build_bloomberg_comdb2]
+    steps:
+      - name: Download bloomberg-comdb2 with build artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: bloomberg-comdb2
+          path: .
+      - name: Install bloomberg-comdb2
+        run: '
+          sudo apt-get install -qy
+            libevent-dev
+            liblz4-dev
+            libprotobuf-c-dev
+            libsqlite3-dev
+            libssl-dev
+            libunwind-dev
+            zlib1g-dev &&
+          tar xvf bloomberg-comdb2.tar.gz &&
+          (cd bloomberg-comdb2/build && sudo make install) &&
+          rm -rf bloomberg-comdb2.tar.gz bloomberg-comdb2/
+        '
+      - uses: actions/checkout@v4
+      - name: 'Set up Python 3.7'
+        uses: actions/setup-python@v4  # note that this step overwrites the PKG_CONFIG_PATH variable
+        with:
+          python-version: '3.7'  # the lowest version that we support in CI
+      - name: Build sdist
+        run: '
+          sudo apt-get install -qy pkg-config &&
+          PKG_CONFIG_PATH=/opt/bb/lib/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig
```

If `actions/setup-python` is setting `PKG_CONFIG_PATH`, we should probably tack on our path to theirs:
```suggestion
          PKG_CONFIG_PATH=/opt/bb/lib/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig:$PKG_CONFIG_PATH
```
I'm assuming `/usr/lib/x86_64-linux-gnu/pkgconfig` is what they're setting it to?

---

Probably. Let's add `$PKG_CONFIG_PATH` at the end yes.